editor/src/core/model/element-metadata-utils.ts

```diff
@@ -794,6 +797,23 @@ export const MetadataUtils = {
       if (!collapsedAncestor) {
         visibleNavigatorTargets.push(path)
       }
+
+      const matchingFocusPath =
+        focusedElementPath == null
+          ? null
+          : TP.staticScenePathContainsElementPath(focusedElementPath, TP.elementPathForPath(path))
+      const focusedRootElementPaths =
+        matchingFocusPath == null
+          ? []
+          : allPaths.filter(
+              (p) =>
+                TP.depth(p) === 2 && // TODO this is actually pretty silly, TP.depth returns depth + 1 for legacy reasons
```

Comment:
It's probably worth extracting the depth check then to a function in `template-path.ts` that checks if the path is a root element path